editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/util/camel/CamelFacetVersionChangeDelegate.java

```diff
@@ -87,13 +87,13 @@ private void updateCamelVersion(IProject project, String camelVersion) throws Co
 		// TODO: this block ensures that we have the staging repo for camel 2.17 in our pom.xml
 		// so we can find that unreleased camel version. this becomes obsolete once the camel version 
 		// has been released and can be disabled / removed / used for a new unreleased camel version 
-		MavenUtils.ensureRepositoryExists(m2m.getRepositories(), PRODUCT_STAGING_REPO_URI, "fuse-ea");
-		MavenUtils.ensureRepositoryExists(m2m.getPluginRepositories(), PRODUCT_STAGING_REPO_URI, "fuse-ea");
-		MavenUtils.ensureRepositoryExists(m2m.getRepositories(), THIRD_PARTY_STAGING_REPO_URI, "redhat-ea-repository");
-		MavenUtils.ensureRepositoryExists(m2m.getPluginRepositories(), THIRD_PARTY_STAGING_REPO_URI, "redhat-ea-repository");
+//		MavenUtils.ensureRepositoryExists(m2m.getRepositories(), PRODUCT_STAGING_REPO_URI, "fuse-ea");
+//		MavenUtils.ensureRepositoryExists(m2m.getPluginRepositories(), PRODUCT_STAGING_REPO_URI, "fuse-ea");
+//		MavenUtils.ensureRepositoryExists(m2m.getRepositories(), THIRD_PARTY_STAGING_REPO_URI, "redhat-ea-repository");
+//		MavenUtils.ensureRepositoryExists(m2m.getPluginRepositories(), THIRD_PARTY_STAGING_REPO_URI, "redhat-ea-repository");
+		// END OF TODO block

```

Comment:
seems a good idea but for 9.0 branch. i would prefer to avoid any not required change in 8.x branch